src/Tickets/Emails/Email/Completed_Order.php

```diff
@@ -7,6 +7,8 @@
 
 namespace TEC\Tickets\Emails\Email;
 
+use Automattic\WooCommerce\StoreApi\Formatters\CurrencyFormatter;
+use TEC\Tickets\Commerce\Utils\Currency;
```

Comment:
Are we using any of these?

Reply:
I was toying around with it, but decided not to use them. Forgot to remove those lines, though. Good catch.